account_move_line_accounting_description_sale/models/sale_order.py

```diff
@@ -0,0 +1,13 @@
+from odoo import models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    def _prepare_invoice_line(self, **optional_values):
+        line = super(SaleOrderLine, self)._prepare_invoice_line(**optional_values)
```

Comment:
```suggestion
        line = super()._prepare_invoice_line(**optional_values)
```
You don't need this in py3 ;)